astropy/units/quantity.py

```diff
@@ -1088,8 +1088,16 @@ def float_formatter(value):
             return Latex.format_exponential_notation(value,
                                                      format_spec=format_spec)
 
+        def complex_formatter(value):
+            return '({0}{1}i)'.format(
+                Latex.format_exponential_notation(value.real,
+                                                     format_spec=format_spec),
```

Comment:
Sorry to be annoying, but could you align the indentation... (here and in line below)

Reply:
Sure, no problem